lib/plugins/orionUrls.js

```diff
@@ -29,68 +29,88 @@
+    ['PUT', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes$/, 'N/A'],
+    ['POST', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes$/, 'N/A'],
+    ['DELETE', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes$/, 'N/A'],
+    ['GET', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes\/.+/, 'read'],
+    ['POST', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes\/.+/, 'create'],
+    ['PUT', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes\/.+/, 'update'],
+    ['DELETE', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes\/.+/, 'delete'],
+    ['GET', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes\/.+\/.+/, 'read'],
+    ['PUT', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes\/.+\/.+/, 'update'],
+    ['DELETE', /^\/(ngsi10|v1)\/contextentities\/.+\/attributes\/.+\/.+/, 'delete'],
+    ['GET', /^\/(ngsi10|v1)\/contextentities\/.+\/attributeDomains\/.+/, 'read'],
+    ['GET', /^\/(ngsi10|v1)\/contextentities\/.+/, 'read'],
+    ['PUT', /^\/(ngsi10|v1)\/contextentities\/.+/, 'update'],
+    ['POST', /^\/(ngsi10|v1)\/contextentities\/.+/, 'create'],
+    ['DELETE', /^\/(ngsi10|v1)\/contextentities\/.+/, 'delete'],
+    ['GET', /^\/(ngsi10|v1)\/contextentitytypes\/.+\/attributes$/, 'N/A'],
+    ['GET', /^\/(ngsi10|v1)\/contextentitytypes\/.+\/attributes\/.+/, 'read'],
+    ['GET', /^\/(ngsi10|v1)\/contextentitytypes\/.+/, 'read'],
+    ['GET', /^\/(ngsi10|v1)\/contextentitytypes\/.+\/attributeDomains\/.+/, 'read'],
+    ['POST', /^\/(ngsi10|v1)\/contextsubscriptions$/, 'subscribe'],
+    ['PUT', /^\/(ngsi10|v1)\/contextsubscriptions\/.+/, 'subscribe'],
+    ['DELETE', /^\/(ngsi10|v1)\/contextsubscriptions\/.+/, 'subscribe'],
+    /* New operations in v1/ (note that they don't use the "ngsi10" alternative in the pattern) */
+    ['GET', /^\/v1\/contextentities/, 'read'],
+    ['POST', /^\/v1\/contextentities/, 'create'],
+    ['GET', /^\/v1\/contextsubscriptions/, 'read'],
+    ['GET', /^\/v1\/contextsubscriptions\/.+/, 'read'],
+    ['GET', /^\/v1\/contexttypes/, 'read'],
+    ['GET', /^\/v1\/contexttypes\/.+/, 'read'],
+
+    /* V2 Operations */
+    ['GET', /^\/v2$/, 'read'],
+    ['GET', /^\/v2\/entities$/, 'read'],
+    ['GET', /^\/v2\/entities\/.+/, 'read'],
+    ['POST', /^\/v2\/entities$/, 'create', {options: 'append'}],
+    ['POST', /^\/v2\/entities$/, 'update'],
+    ['PATCH', /^\/v2\/entities\/.+/, 'update'],
+    ['POST', /^\/v2\/entities\/.+/, 'update'],
+    ['DELETE', /^\/v2\/entities\/.+/, 'delete'],
+    ['PUT', /^\/v2\/entities\/.+/, 'update'],
+    ['GET', /^\/v2\/entities\/.+\/attrs\/.+/, 'read'],
+    ['PUT', /^\/v2\/entities\/.+\/attrs\/.+/, 'update'],
+    ['DELETE', /^\/v2\/entities\/.+\/attrs\/.+/, 'delete'],
+    ['GET', /^\/v2\/entities\/.+\/attrs\/.+\/value$/, 'read'],
+    ['PUT', /^\/v2\/entities\/.+\/attrs\/.+\/value$/, 'update'],
+    ['GET', /^\/v2\/types$/, 'read'],
+    ['GET', /^\/v2\/types\/.+/, 'read'],
+    ['GET', /^\/v2\/subscriptions$/, 'read'],
+    ['GET', /^\/v2\/subscriptions\/.+/, 'read']
```

Comment:
Being strict, #259 specifies "to DELETE /v2/subscriptions/<subId>", trying to "save time" in the future adding the rest of CRUD operations for subscriptions although currently they are not implemented. However, we can re-define the scope of #259 if you prefer to keep the code as it is now.